test/unit/typography/p5.Font.js

```diff
@@ -0,0 +1,28 @@
+
+suite('p5.Font', function() {
+
+  var v, myp5 = new p5(function(p) {
+    p.setup = function() {};
+    p.draw = function() {};
+  });
+
+  suite('p5.prototype.p5.Font()', function() {
+    setup(function() {
+      v = new p5.Font(myp5);
```

Comment:
This is a comment about the API for `p5.Font`, not the test itself: I'm surprised we need to pass a p5 instance as a parameter when constructing a Font object. What motivated that decision?